src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -471,7 +471,7 @@ private static void getFormat(StreamType streamType, DLNAMediaInfo media, DLNAMe
 			format = FormatConfiguration.MOV;
 		} else if (
 				value.contains("isom") ||
-				value.startsWith("mp4") ||
+				(streamType != StreamType.Audio && value.startsWith("mp4")) ||
```

Comment:
It will still leave you with some other issue, like MP4 SLS

Reply:
@Sami32 I didn't quite understand, are you saying that this breaks SLS?

Reply:
Just that it will fail on some format and still have a bad, incorrect parsing.